docs/2023/AI Tools/Document Uploader/updates/2023-08-04.md

```diff
@@ -4,13 +4,11 @@ author: Harshit Gupta
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
-
-## Screenshots / Videos 
+- [ ] Creating a Efficinet mechanism for evaluating metrics for the given models.
+- [ ] Implement Langchain chunking in AI Tools.
 
 ## Contributions
+Searching methods to implement evaluation metrics efficiently by a given config. file.
```

Comment:
No need of adding this, Better to remove this, approving the PR as it is a minor comment